projects/fsastorefrontlib/src/cms-components/checkout/components/quote-review/quote-review.component.ts

```diff
@@ -54,9 +51,38 @@ export class QuoteReviewComponent implements OnInit, OnDestroy {
     protected translationService: FSTranslationService,
     protected checkoutService: FSCheckoutService,
     protected globalMessageService: GlobalMessageService,
+    protected consentConnector: ConsentConnector,
+    protected userAccountFacade: UserAccountFacade,
+    protected oboConsentService: ConsentService,
     protected winRef?: WindowRef
   ) {}
 
+  cart$: Observable<Cart>;
+  showContent$: Observable<boolean> = of(true);
+  isCartStable$: Observable<boolean>;
+  subscription = new Subscription();
+  modalRef: ModalRef;
+  cartCode: string;
+  previousCheckoutStep$: Observable<FSSteps>;
+  nextCheckoutStep$: Observable<FSSteps>;
+  activeCategory$: Observable<string>;
+  baseUrl: string;
+  selectedIndex = -1;
+
+  oboCustomers$: Observable<
+    OBOCustomerList
+  > = this.userAccountFacade.get().pipe(
+    filter(user => !!user && user.roles.includes(FSUserRole.SELLER)),
+    take(1),
+    switchMap(user => this.consentConnector.getOBOCustomerList(user.uid))
+  );
+
+  isCartTransferAllowedForSeller$: Observable<
+    boolean
+  > = this.oboConsentService
+    .isCartTransferAllowedForSeller()
+    .pipe(shareReplay());
```

Comment:
Used `shareReplay` here to prevent multiple evaluations from executing in the template.

Reply:
I see now there is no need to change something in the template since you use `shareReplay`